src/components/JobLogs/withJobLogsCellsFormatter.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  injectIntl,
+  intlShape,
+} from 'react-intl';
+
+import { compose } from '../../utils';
+
+const withJobLogsCellsFormatter = WrappedComponent => {
+  return class extends Component {
+    static propTypes = {
+      intl: intlShape.isRequired,
+      formatter: PropTypes.object,
+    };
+
+    constructor(props) {
+      super(props);
+
+      this.cellFormatters = {
+        ...props.formatter,
+        runBy: this.formatUser,
+        completedDate: this.formatEndedRunningDate,
+      };
+    }
+
+    formatUser({
+      runBy: {
```

Comment:
Please use class property approach. Also in this case I suggest to move destructuring from declaration to function body.

Reply:
Do you mean to use static?